### ISSUE TITLE
installation path colum for installed packages

### ISSUE THREAD

**Issue by Vaquero84** [chocolatey/ChocolateyGUI#278]
With chocolatey I'm not always sure where packages, especially with natice installers, get installed to. So it would be great if there was a column showing the installation path in the "This PC" category. In addition to that a context menu with "show in explorer" would be helpful, too.

**Comment by gep13** [comment]
I don't think that this is something that Chocolatey has knowledge of either.  @ferventcoder is this something that can be exposed?

**Comment by Vaquero84** [comment]
At least for some packages with installers I found the installation path by looking inside the "\chocolatey\.chocolatey\packagename.version\.registry" (if there is one) for <InstallLocation>. So I guess it could be exposed. Unfortunately for some MSI packages I only get a cryptic key which is the id i presume. It also tells you the installertype like "MSI", "InnoSetup", "Unknown" or "Custom".
Well, as for zip packeges, maybe the .files or the chocolatey.log is any good?

**Comment by ferventcoder** [comment]
Yes, choco has knowledge of where it installs given the registry snapshot gathers that information.

**Comment by gep13** [comment]
@ferventcoder interesting... so this could be something that we expose in ChocolateyGUI. :+1:

**Comment by RichiCoder1** [comment]
I don't think this is something we'd get around to for a while, but if someone else would be interested, go for it.

**Comment by gep13** [comment]
This is related to this issue: https://github.com/chocolatey/ChocolateyGUI/issues/597

One is for a change to the package details page, and the other for list view.

**Comment by Mugane** [comment]
Am I correct in my assessment that after ten years this has yet to be done?
